File: native-app/native-app/Sources/Models/ObjectClassification.swift
//
//  ObjectClassification.swift
//  native-app
//
//  Created by Michael Pavkovic on 6/8/20.
//  Copyright © 2020 Unity Technologies. All rights reserved.
//

// These models are unused since all processing was moved to Unity


struct ObjectClassification: Codable {
    
    let label: String
    let box: BoundingBox
    let score: Float
    
}

File: unity-component/Assets/Scripts/Models/ObjectClassification.cs
namespace Models
{
    [System.Serializable]
    public class ObjectClassification
    {
        public string label;
        public int label_id;
        public BoundingBox box;
        public float score;

        public ObjectClassification(string label, int labelId, BoundingBox box, float score)
        {
            this.label = label;
            label_id = labelId;
            this.box = box;
            this.score = score;
        }
    }
}